net_transport.go

```diff
@@ -255,8 +259,7 @@ func (n *NetworkTransport) getProviderAddressOrFallback(id ServerID, target Serv
 	if n.serverAddressProvider != nil {
 		serverAddressOverride, err := n.serverAddressProvider.ServerAddr(id)
 		if err != nil {
-			n.logger.Printf("[WARN] Unable to get address for server id %v, got error:%v", id, err)
-			n.logger.Printf("[INFO] Using fallback address %v", target)
+			n.logger.Printf("[WARN] Unable to get address for server id %v, got error:%v. Using fallback address", id, err, target)
```

Comment:
The args don't match the format string for this one, and i'd put the error at the end since it could be anything, so:

`n.logger.Printf("[WARN] Unable to get address for server id %v, using fallback address %v: %v", id, target, err)`